atd-vze/src/views/VZDashboard/VZDashboard.js

```diff
@@ -0,0 +1,69 @@
+import React from "react";
+import { Col, Row } from "reactstrap";
+import { withApollo } from "react-apollo";
+import { useQuery } from "@apollo/react-hooks";
+import Widget02 from "../Widgets/Widget02";
+
+import { GET_CRASH } from "../../queries/dashboard";
+
+function VZDashboard() {
+  const year = new Date().getFullYear();
+  const yearStart = `${year}-01-01`;
+  const yearEnd = `${year}-12-31`;
+  const { loading, error, data } = useQuery(GET_CRASH, {
+    variables: { yearStart, yearEnd },
+  });
+
+  if (loading) return "Loading...";
+  if (error) return `Error! ${error.message}`;
+
+  const yearsOfLifeLostYTD =
+    data.atd_txdot_primaryperson_aggregate.aggregate.sum.years_of_life_lost +
+    data.atd_txdot_person_aggregate.aggregate.sum.years_of_life_lost;
+  const fatalitiesYTD =
+    data.atd_txdot_crashes_aggregate.aggregate.sum.death_cnt;
+  const seriousInjuriesYTD =
+    data.atd_txdot_crashes_aggregate.aggregate.sum.sus_serious_injry_cnt;
+  const crashesYTD = data.atd_txdot_crashes_aggregate.aggregate.count;
+
+  return (
+    <div className="animated fadeIn">
+      <Row>
+        <Col xs="12" sm="6" md="4">
+          <Widget02
+            header={yearsOfLifeLostYTD + ""}
```

Comment:
I think it would be nicer to use template literal interpolation syntax here instead of `+ year` and the `+ ""` string conversion trick here and for all the component props.

```js
header={`${yearsOfLifeLostYTD}`}
...
mainText={`Years of life lost in ${year}`}

```

Reply:
Done!